File: lab/javascript/tecnicas/swap.js
let a = 5;
let b = 10;
console.log("Antes:", a, b);

// usando temp
let temp = a;
a = b;
b = temp;
console.log("Depois (com temp):", a, b);

// Jstonic (sem temp)
[a, b] = [b, a];
console.log("Depois (Pythonic):", a, b);

File: lab/python/tecnicas/swap.py
a, b = 5, 10
print("Antes:", a, b)

# usando temp
temp = a
a = b
b = temp
print("Depois (com temp):", a, b)

# Pythonic (sem temp)
a, b = b, a
print("Depois (Pythonic):", a, b)
